### PR TITLE
Add section on how to install Helm on Docker EE

### PR DESCRIPTION
**Pull request by traci-morrison**
Signed-off-by: Traci Morrison <traci.morrison@docker.com>

### Proposed changes

Adding section on how to install Helm on Docker EE
### Related issues (optional)
https://docker.atlassian.net/browse/ENGDOCS-33

### CODE REVIEW

File: ee/ucp/user-access/kubectl.md [hunk 1]
```diff
@@ -94,11 +94,62 @@ curl https://storage.googleapis.com/kubernetes-release/release/$env:k8sversion/b
...
+This section describes how to grant the correct service account/permissions on tiller to install applications into the cluster.
+
+### Prerequisites
+Before performing these steps, you must meet the following requirements:
+- You must be running a Docker EE 2.1 or higher cluster
+- You must have kubectl configured to communicate with the cluster (usually this is done via a client bundle)
+
+To create a service account for tiller and apply the correct grants:
+
+1. Create the tiller service account:
```
Comment (usha-mandya) [inline ee/ucp/user-access/kubectl.md]: Nit: Should this be  '...Tiller service..'?

File: ee/ucp/user-access/kubectl.md [hunk 2]
```diff
@@ -94,11 +94,62 @@ curl https://storage.googleapis.com/kubernetes-release/release/$env:k8sversion/b
...
+
+5. In the Resource Set area, move the slider to enable **Apply Role Binding to all namespace (Cluster Role Binding)**.
+
+6. Click **Next**.
+
+7. In the Role area, select **cluster-admin**.
+
+8. Click **Create**.
+
+9.  At this stage, if you have tiller installed in the cluster already, you need to patch the tiller deployment to use the tiller service account you created:
```
Comment (usha-mandya) [inline ee/ucp/user-access/kubectl.md]: if you have Tiller installed?

**Comment by usha-mandya** [comment]
@traci-morrison - Looks good overall. Noted a few nits about the capitalization for Tiller. 

Also, the numbering gets a bit funky due to the embedded code samples - https://deploy-preview-9222--docsdocker.netlify.com/ee/ucp/user-access/kubectl/#prerequisites. Perhaps indenting them will fix the numbering issue?

**Comment by traci-morrison** [comment]
Note to self: Add a link to this section here: https://docs.docker.com/ee/ucp/admin/configure/enable-helm-tiller/

**Comment by traci-morrison** [comment]
Closing in favor of https://github.com/docker/docker.github.io/pull/9257

**Pull request closed without merging**